cg/cli/deliver/base.py

```diff
@@ -22,6 +24,7 @@ def deliver(context):
     LOG.info("Running CG deliver")
     context.obj["status_db"] = Store(context.obj["database"])
     context.obj["housekeeper_api"] = HousekeeperAPI(context.obj)
+    context.obj["rsync_api"] = RsyncAPI(context.obj)
```

Comment:
do we need this ?

Reply:
I think you are right, this line might not be needed. I will test without it and then remove if not needed

Reply:
I removed that line now and everything still works as it should when I test on stage